apps/src/lib/kits/maker/ui/DiscountAdminOverride.jsx

```diff
@@ -1,142 +1,255 @@
+                `${this.state.discountCode} (${this.state.fullDiscount ? 'FULL' : 'PARTIAL'})`
+              }
+            </div>
+            <div>Current override: {this.state.adminOverride}</div>
             <h4>Option 1: Link teacher account with other accounts</h4>
             <div>
               If teacher meets the eligibity requirements but is simply using a
                different email address for their account than what we have on file,
                please go to the <a href="https://studio.code.org/admin/studio_person">
                Studio Person ID admin page </a> to link this acccount
-               to the saccount associated with the email address we have on file.
+               to the account associated with the email address we have on file.
             </div>
             <h4>Option 2: Give teacher a discount code</h4>
             <div>
               If the teacher has already received their code and it was for the incorrect amount,
               email <a href="mailto:adaaccounts@adafruit.com"> adaaccounts@adafruit.com </a>
               so they can disable that code first. We should not be doing this override if the
               teacher has already used the incorrect code to purchase a kit.
-              <form>
-                <label>
-                  <input
-                    type="radio"
-                    name="discountAmount"
-                    value="full"
-                  />
-                  Teacher should receive 100% discount code (kit price would become $0)
-                </label>
-                <label>
-                  <input
-                    type="radio"
-                    name="discountAmount"
-                    value="partial"
-                  />
-                  Teacher should receive partial discount code (kit price would become $97.50)
-                </label>
-                <Button
-                  color={Button.ButtonColor.orange}
-                  text="Submit"
-                  onClick={() => {}}
+            </div>
+            <div style={styles.radioContainer}>
+              <label>
+                <input
+                  style={styles.radio}
+                  type="radio"
+                  name="discountAmount"
+                  value="full"
+                  checked={this.state.overrideValue === "full"}
+                  onChange={this.handleOverrideChange}
                 />
-              </form>
+                Teacher should receive 100% discount code (kit price would become $0)
+              </label>
+              <label>
+                <input
+                  style={styles.radio}
+                  type="radio"
+                  name="discountAmount"
+                  value="partial"
+                  checked={this.state.overrideValue === "partial"}
+                  onChange={this.handleOverrideChange}
+                />
+                Teacher should receive partial discount code (kit price would become $97.50)
```

Comment:
I've been struggling a little bit with exactly how powerful this page needs to be. The truth is, using this page should already be the exceptional case (though I'm sure there will be a fair number of places where it is necessary). I think I'm okay with saying that if we need additional functionality, it will require manual dev intervention, rather than trying to predict in advance all the additional scenarios we might hit.